flows/waits/flow.go

```diff
@@ -14,25 +13,6 @@ type FlowWait struct {
 	FlowUUID flows.FlowUUID `json:"flow_uuid"   validate:"required,uuid4"`
 }
 
-func (w *FlowWait) Resolve(key string) interface{} {
```

Comment:
We don't currently expose waits in the context. Not it makes sense to either since if there's an active wait, we're not in the engine by definition.